src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    public Square(String color, String name) {
+        setColor(color);
+        setName(name);

```

Comment:
The constructor should accept dimensions of the square (e.g., side length) instead of just color and name. These dimensions should be used in the getArea() method.